queries/javascript/locals.scm

```diff
@@ -28,6 +28,9 @@
 (variable_declarator
   name: (identifier) @definition)
 
+(import_specifier
+  (identifier) @definition)
+
```

Comment:
i don't think this is needed in this PR :)

Reply:
Agreed. I'll remove it and submit another PR for it.